central/reports/common/report_formatter.go

```diff
@@ -89,8 +90,12 @@ type ZippedCSVResult struct {
 
 // Format takes in the results of vuln report query, converts to CSV and returns zipped CSV data and
 // a flag if the report is empty or not. For v1 config pass an empty string.
-func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult, configName string) (*ZippedCSVResult, error) {
-	csvWriter := csv.NewGenericWriter(csvHeader, true)
+func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult, configName string, includeNvd bool) (*ZippedCSVResult, error) {
+	csvHeaderRep := csvHeader
```

Comment:
Not sure how go does it. Does this assignment create a copy of `csvHeader` ? Or would updates to `csvHeaderRep` still get applied to it?

Reply:
I have tested this. Go will create a new copy of csvheader